R/mod_tabpanel_input.R

```diff
@@ -202,7 +202,7 @@ mod_tabpanel_input_server <- function(id, data, errors_detected) {
     filter_var_opts <- shiny::reactive({
       shiny::req(available_var_opts())
       shiny::req(data_sub())
-      date_opts <- intersect(names(data_sub()), c("date_report"))
+      date_opts <- intersect(names(data_sub()), c("date_report", "age"))
```

Comment:
to be very consistent here with naming acan you remove "age here? Because these are supposed to be really the date variables. It is good you did not add it to available_var_opts() as we do not want to allow to also stratify by age. Thus maybe you can just add a seperate intersect of age with the dataset here and then add it to the all_opts